cumulus/parachains/runtimes/assets/common/src/lib.rs

```diff
@@ -111,7 +111,15 @@ pub type ForeignAssetsConvertedConcreteId<AdditionalLocationExclusionFilter, Bal
 		Balance,
 	>;
 
-type AssetIdForPoolAssets = u32;
+pub type AssetIdForPoolAssets = u32;
+/// `Location` vs `AssetIdForPoolAssets` converter for `PoolAssets` with explicit `v3 Location`.
+pub type AssetIdForPoolAssetsConvertV3Location<PoolAssetsPalletLocation> = AsPrefixedGeneralIndex<
+	PoolAssetsPalletLocation,
+	AssetIdForPoolAssets,
+	JustTry,
+	xcm::v3::Location,
```

Comment:
@liamaharon please, can you check (I mean review, one more missing :) ) this: https://github.com/paritytech/polkadot-sdk/pull/4037, I would like to merge it before, because I am trying to avoid using particular XCM version e.g. `xcm::v3:` here in `assets-common`